README.md

```diff
@@ -43,6 +43,18 @@ Supporting built-in subclassing has also negatively affected authoring of new bu
 
 Type I is supported if creating subclasses of built-ins is possible. For example, if derived class can call `super()`. Support for this is provided via `new.target`.
 
+### Example
+
+```js
+class A extends Array {
+  constructor() {
+    super();
+  }
+}
+A.from([1,2,3])     // return type: Array
```

Comment:
The example should probably be more like `new A // return type: A` to show that `new.target` is propagated and determines the [[Prototype]] of the created instance.

Reply:
Don't know if i got this right now...